contrib/inventory/gce.py

```diff
@@ -272,13 +275,17 @@ def get_gce_driver(self):
         # exists.
         secrets_path = self.config.get('gce', 'libcloud_secrets')
         secrets_found = False
-        try:
-            import secrets
-            args = list(getattr(secrets, 'GCE_PARAMS', []))
-            kwargs = getattr(secrets, 'GCE_KEYWORD_PARAMS', {})
-            secrets_found = True
-        except:
-            pass
+
+        # check if secrets exist in the current directory, to avoid
+        # loading the python 3 module 'secrets'
+        if os.path.exists('secrets.py'):
```

Comment:
+1 erjohnso. Removing getattr will solve the issue (because try/except will ignore the builtin 'secrets' module because GCE_PARAMS and GCE_KEYWORD_PARAMS are not declared).